test/parallel/test-fs-read.js

```diff
@@ -53,3 +53,15 @@ test(Buffer.allocUnsafe(expected.length),
 test(new Uint8Array(expected.length),
      new Uint8Array(expected.length),
      Uint8Array.from(expected));
+
+{
+  // Reading > 4gb should return no data
```

Comment:
This code comment is a little confusing because it's merely reading past the end of file that results in no data. That can happen no matter the file size.

Perhaps it should instead mention that it's testing whether >32-bit position arguments read from the absolute (good) or current (bad) position and that reading beyond the file is what should return no data.

Reply:
tweaked the comment